src/Compilers/Core/Portable/CodeAnalysis.csproj

```diff
@@ -816,6 +808,7 @@
     <Compile Include="Xml\XmlCharType.cs" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.DiaSymReader.Native" Version="$(MicrosoftDiaSymReaderNativeVersion)" />
```

Comment:
Seems to work for unit tests, but not for VSIX. Not sure why. I'll try something else.